rdagent/components/coder/data_science/ensemble/eval.py

```diff
@@ -0,0 +1,65 @@
+import json
+from dataclasses import dataclass
+from pathlib import Path
+import numpy as np
+
+from rdagent.components.coder.CoSTEER.evaluators import (
+    CoSTEEREvaluator,
+    CoSTEERSingleFeedback,
+)
+from rdagent.core.evaluation import Feedback
+from rdagent.core.evolving_framework import QueriedKnowledge
+from rdagent.core.experiment import FBWorkspace, Task
+from rdagent.oai.llm_utils import APIBackend
+from rdagent.utils.agent.tpl import T
+from rdagent.utils.env import DSDockerConf, DockerEnv
+
+DIRNAME = Path(__file__).absolute().resolve().parent
+
+@dataclass
+class EnsembleEvalFeedback(Feedback):
```

Comment:
Use the more general version provided by bowen

Reply:
This is now resolved.